packages/manager/src/index.ts

```diff
@@ -0,0 +1,27 @@
+// This file defines the root `@slicemachine/manager` export.
+// Non-Node.js-compatible exports should be defined in `./client.ts`.
```

Comment:
The reverse was detailed in this comment: https://github.com/prismicio/slice-machine/pull/799#discussion_r1115147894

This file can contain exports that use Node.js APIs. Most `@slicemachine/manager` code will be exported from this file since manager uses APIs like `fs` heavily.